devolutions-session/Cargo.toml

```
@@ -8,31 +8,53 @@ description = "Session host application for Devolutions Agent"
 build = "build.rs"
 publish = false
 
+[lints]
+workspace = true
+
+[features]
+default = ["dvc"]
+dvc = [
+    "async-trait",
+    "windows",
+    "now-proto-pdu",
+    "ironrdp",
+    "tempfile",
+    "thiserror",
+    "win-api-wrappers",
```

suggestion: Use `dep:` prefix for dependencies.
```suggestion
    "dep:async-trait",
    "dep:windows",
    "dep:now-proto-pdu",
    "dep:ironrdp",
    "dep:tempfile",
    "dep:thiserror",
    "dep:win-api-wrappers",
```